spec/index.html

```diff
@@ -568,13 +568,13 @@ <h3>RDF Blank Nodes</h3>
   <section id="unlabeled-bnodes">
```

Comment:
```suggestion
  <section id="nesting-of-bnodes">
```

Reply:
Can't really change fragment identifiers for sections which were present in RDF 1.1, which could create a referencing problem. There are other cases where the section identifiers are somewhat out of date, but they don't convey any meaning in and of themselves.

Reply:
Can the same trick as with other obsolete terms/links not be used? Let the old links still reach the desired section, but aim toward modern fragment identifiers (which many people still treat as meaning something, even if they aren't meant to)?

Reply:
Yes, we could add `<span id="unlabeled-nodes"></span>` and change this identifier, but there are many such identifiers that could be updated, and groups in the past have avoided doing this. It's somewhat different from terms, for which we have tried to maintain references, but in that case it's really necessary to update the terminology. Updating the identifier for a section where the section name has changed doesn't seem to be at the same level, but I'm open to doing this more comprehensively, if the group wishes to do so.

Reply:
I understand the issue and I am fine to keep it as is for the moment.